src/System.Runtime.WindowsRuntime/src/project.json

```diff
@@ -8,6 +8,12 @@
       "imports": [
         "dotnet5.4"
       ]
+    },
+    "uap10.1": {
```

Comment:
deleted this file. I don't see it reference netcore50aot\project.json  in .csproj file